Traceback (most recent call last):
  File "/home/runner/work/semgrep/semgrep/cli/bin/semgrep", line 174, in <module>
    exec_osemgrep()
  File "/home/runner/work/semgrep/semgrep/cli/bin/semgrep", line 155, in exec_osemgrep
    os.execvp(str(path), sys.argv)
  File "/usr/lib/python3.10/os.py", line 575, in execvp
    _execvpe(file, args)
  File "/usr/lib/python3.10/os.py", line 598, in _execvpe
    exec_func(file, *argrest)
FileNotFoundError: [Errno 2] No such file or directory
